src/controllers/CheckYourAppealController.ts

```diff
@@ -26,7 +34,38 @@ export class CheckYourAppealController extends BaseAsyncHttpController {
             .orDefault({});
 
         return this.render('check-your-appeal', { ...appealsData, userProfile });
+    }
+
+    @httpPost('')
+    public async handleFormSubmission(req: Request): Promise<HttpResponseMessage> {
+        const userProfile = req.session
+            .chain(_ => _.getValue<ISignInInfo>(SessionKey.SignInInfo))
+            .map(info => info[SignInInfoKeys.UserProfile])
+            .extract() as IUserProfile;
+
+        const appealsData = req.session
+            .chain(_ => _.getExtraData())
+            .chain(data => Maybe.fromNullable(data[AppealKeys.APPEALS_KEY]))
+            .extract() as Appeal;
 
+        try {
+            await this.emailService.send({
+                to: userProfile.email as string,
+                subject: 'Your appeal has been submitted',
+                body: {
+                    templateName: 'lfp-appeal-submission-confirmation',
+                    templateData: {
+                        companyNumber: appealsData[AppealKeys.PENALTY_IDENTIFIER][PenaltyIdentifierKeys.COMPANY_NUMBER],
+                        userProfile: {
+                            email: userProfile.email
+                        }
+                    }
+                }
+            })
+        } catch (err) {
```

Comment:
I need to double check it with Bart. It feels like confirmation email is less critical and for that reason I decided to render no error message when sending failed.

Reply:
From a developers point of view, isn't it handy for us to throw a proper error so that we can essentially fail early if something goes wrong?

Reply:
Double checked with Bart and agreed that in first version any problem will result in an error page being displayed. That will get improved in the future based on the outcome of white-board session.